python/ray/tune/tests/test_tune_restore.py

```diff
@@ -64,7 +67,7 @@ def testTuneRestore(self):
             name="TuneRestoreTest",
             stop={"training_iteration": 2},  # train one more iteration.
             checkpoint_config=CheckpointConfig(checkpoint_frequency=1),
-            restore=self.checkpoint_path,  # Restore the checkpoint
+            restore=self.checkpoint_parent,  # Restore the checkpoint
```

Comment:
@justinvyu to confirm that this is the intended behavior - `restore` takes in the directory instead of the file? Seems logical, but are there backwards compatibility issues with this?